docs/changelog/104269.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 104269
+summary: "ESQL: Support loading points from source into WKB blocks"
+area: "Geo"
```

Comment:
All previous PRs I created used area "ES|QL", but perhaps if most future Geo work will be in ESQL, we should use area "Geo"?